nextcord/guild.py

```diff
@@ -520,6 +522,8 @@ def _from_data(self, guild: GuildPayload) -> None:
             "premium_progress_bar_enabled"
         )
 
+        self.safety_alerts_channel_id: Optional[int] = guild.get("safety_alerts_channel_id")
```

Comment:
There should be a property, `safety_alerts_channel`, that returns the underlying channel object this id represents. Similar to `system_channel`, `rules_channel`, and `public_updates_channel`.

Reply:
@spifory did you forget to push your commits? This was marked as resolved without anything.

Even if this was kept as an attribute, theres no documentation.

Reply:
Yep I had forgotten to push my commits, I've pushed them now.